lizard_downloader.py

```diff
@@ -230,7 +231,17 @@ def show_data(self):
 
         # Create a new memory vector layer
         self.layer = QgsVectorLayer(
-            "{}?crs={}".format(geometry_type, WGS84), asset_types[0], "memory")
+            "{}?crs={}".format(geometry_type, wgs84), asset_types[0], "memory")
+
+        # Add Lizard style (SVG)
```

Comment:
I would refactor this code block into a function and move it to a `utils` module. Then import it and call it here. That way other modules can also use this general functionality.

Write a function called `apply_style(layer, asset_type)` in a module `styler` in the `utils` module, much like the 3di qgis plugin does. Pass a layer and an asset_type et voila!